src/commands/android/subcommands/uninstall/system-image.ts

```diff
@@ -0,0 +1,94 @@
+import colors from 'ansi-colors';
+import inquirer from 'inquirer';
+
+import Logger from '../../../../logger';
+import {Platform} from '../../interfaces';
+import {execBinarySync} from '../../utils/sdk';
+import {getBinaryLocation} from '../../utils/common';
+import {getInstalledSystemImages, showMissingBinaryHelp} from '../common';
+
+export async function deleteSystemImage(sdkRoot: string, platform: Platform): Promise<boolean> {
+  try {
+    const sdkmanagerLocation = getBinaryLocation(sdkRoot, platform, 'sdkmanager', true);
+    if (!sdkmanagerLocation) {
+      showMissingBinaryHelp('sdkmanager');
+
+      return false;
+    }
+    const installedImages: string[] = await getInstalledSystemImages(sdkmanagerLocation, platform);
+    if (!installedImages.length) {
+      return false;
+    }
+
+    const systemImageAnswer = await inquirer.prompt({
+      type: 'list',
+      name: 'systemImage',
+      message: 'Select the system image to uninstall:',
+      choices: installedImages
+    });
+    const systemImage = systemImageAnswer.systemImage;
+
+    Logger.log();
+    Logger.log(`Uninstalling ${colors.cyan(systemImageAnswer.systemImage)}...\n`);
+
+    const deleteStatus = execBinarySync(sdkmanagerLocation, 'sdkmanager', platform, `--uninstall '${systemImage}'`);
+    if (!deleteStatus?.includes('100% Fetch remote repository')) {
+      Logger.log(`${colors.red('Failed to uninstall system image!')} Please try again.`);
+
+      return false;
+    }
+    Logger.log(colors.green('System image uninstalled successfully!\n'));
+
+    deleteObsoleteAVDs(sdkRoot, platform);
```

Comment:
We should ask before deleting these because if the system image is downloaded back, they will also start to work again.

Reply:
Ask the question after listing out all the obsolete AVDs.